sources/Metal/Infrastructure/iTermMetalRenderer.h

```diff
@@ -46,6 +46,7 @@ NS_CLASS_AVAILABLE(10_11, NA)
 @interface iTermMetalRendererTransientState : NSObject
 @property (nonatomic, strong, readonly) __kindof iTermRenderConfiguration *configuration;
 @property (nonatomic, strong) id<MTLBuffer> vertexBuffer;
+@property (nonatomic, strong) id<MTLBuffer> verticalGuideVertexBuffer;
```

Comment:
You shouldn't need to add another vertex buffer in here. This class is shared by all the renderers. The `vertexBuffer` property is there as a convenience for the common case of only having one. If a renderer needs multiple vertex buffers it should handle them itself. Look at iTermIndicatorRenderer.m for an example.

Reply:
Done (removed).